game_engine/ecs/systems/ServerInputSystem.hpp

```
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <chrono>
+#include <unordered_map>
+#include "ECSManager.hpp"
+#include "Packet.hpp"
+#include "Queue.hpp"
+#include "System.hpp"
+
+namespace ecs {
+  struct PlayerInput {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: input, sequence_number, timestamp

(cppcoreguidelines-pro-type-member-init)